fastlane-ios/fastlane/Fastfile

```diff
@@ -0,0 +1,40 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
+
+platform :ios do
+  
+  project_path = "myProject/myProject.xcodeproj"
+
+  desc "Run iOS tests"
+  lane :test do
+    scan(
```

Comment:
```suggestion
    run_tests(
```

Scan is now the older/original way and an alias of `run_tests` https://docs.fastlane.tools/actions/run_tests/